documents/design/d_star_lite.md

```diff
@@ -0,0 +1,105 @@
+# Project Title
+
+*Issue #567*
+(What GitHub issue is associated with this project)
+
+**Author:**
+- Christopher Lo
+
+## The Problem
+
+What is the problem you are trying to solve? Why is it important to solve this problem?
+What is the end goal of this project? This section should similar to the body of the GitHub
+issue but include more details about it. This will be a qualitative description of the
+project. Use as many details you can to describe the requirements of the project.
+*(This is similar to the Motivation section of the other design documents.)*
```

Comment:
You should remove all the parts that are like the "guide paragraphs" to keep the document short and concise.